config/.env.default

```diff
@@ -0,0 +1,23 @@
+NODE_ENV=
```

Comment:
In Marmelab project, we prefer to name `.env.dist`

Reply:
With `cp -n ./config/.env.dist ./config/.env` inside `install` action  on Makefile. Can do in other PR